tendrl/node_agent/collectd/collectors/gluster/utils.py

```diff
@@ -0,0 +1,273 @@
+import collectd
+import ConfigParser
+import os
+import shlex
+import socket
+import subprocess
+from subprocess import Popen
+import time
+import traceback
+import ini2json
+
+
+GLUSTERD_ERROR_MSG = 'Connection failed. Please check if gluster daemon is operational.'
+
+
+def get_brick_state_mapping(status):
+    return {
+        'Started': 0,
+        'Stopped': 2
+    }.get(status, 1)
+
+
+def get_volume_state_mapping(status):
+    return {
+        'Started': 0,
+        'Stopped': 2
+    }.get(status, 1)
+
+
+def exec_command(cmd_str):
```

Comment:
If this is called from the node-agent service and not from collectd service then use commons.cmd_utils